<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>662</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>372</width>
    <height>662</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>372</width>
    <height>662</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Lista de Contatos</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #fff</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="gerenciadorTelas">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>371</width>
      <height>661</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="telaInicial">
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>371</width>
        <height>591</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QListWidget" name="listWidget">
         <property name="styleSheet">
          <string notr="true">border: none;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btnAddContact">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>600</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #48A7FF;
	border-radius: 20%;
}

QPushButton:hover{
	background-color: #4DBCFF;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="src.qrc">
        <normaloff>:/src/img/addButton.png</normaloff>:/src/img/addButton.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSearchContact">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>20</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: none;
	border: none;
}

QPushButton:hover{
	background-color: #E1E1E1;
	border-radius: 7px;
}</string>
      </property>
      <property name="text">
       <string>üîç</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inptSearchInput">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>351</width>
        <height>41</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background-color: #F0F0F0;
	padding-left: 18px;
	padding-right: 18px;
	border:none;
	padding-bottom: 1px;
}

QLineEdit:focus{
	border-bottom: 1px solid rgb(208, 208, 208) ;
	padding-bottom: 0;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Pesquise pelo nome...</string>
      </property>
     </widget>
     <zorder>verticalLayoutWidget_2</zorder>
     <zorder>btnAddContact</zorder>
     <zorder>inptSearchInput</zorder>
     <zorder>btnSearchContact</zorder>
    </widget>
    <widget class="QWidget" name="CRUD">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="lblTitulo">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>371</width>
        <height>61</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>375</width>
        <height>62</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>375</width>
        <height>62</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>375</width>
        <height>62</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #F0F0F0;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#0e0e0e;&quot;&gt;Titulo legal&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnBack">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: none;
	border: none;
}

QPushButton:hover{
	background-color: #E1E1E1;
	border-radius: 7px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="src.qrc">
        <normaloff>:/src/img/iconBack.png</normaloff>:/src/img/iconBack.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>213</width>
        <height>213</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>213</width>
        <height>213</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>213</width>
        <height>213</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>213</width>
        <height>213</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="src.qrc">:/src/img/iconPerson213.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="inptNome">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>355</width>
        <height>51</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>355</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>355</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background-color: #F0F0F0;
	padding-left: 18px;
	padding-right: 18px;
	border:none;
	padding-bottom: 1px;
}

QLineEdit:focus{
	border-bottom: 1px solid rgb(208, 208, 208) ;
	padding-bottom: 0;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="inptDia">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>460</y>
        <width>163</width>
        <height>51</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>163</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>163</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background-color: #F0F0F0;
	padding-left: 18px;
	padding-right: 18px;
	border:none;
	padding-bottom: 1px;
}

QLineEdit:focus{
	border-bottom: 1px solid rgb(208, 208, 208) ;
	padding-bottom: 0;
}</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="inptMes">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>460</y>
        <width>163</width>
        <height>51</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>163</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>163</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background-color: #F0F0F0;
	padding-left: 18px;
	padding-right: 18px;
	border:none;
	padding-bottom: 1px;
}

QLineEdit:focus{
	border-bottom: 1px solid rgb(208, 208, 208) ;
	padding-bottom: 0;
}</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="cursorMoveStyle">
       <enum>Qt::LogicalMoveStyle</enum>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>351</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #6A6A6A;</string>
      </property>
      <property name="text">
       <string>Nome</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>440</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #6A6A6A;</string>
      </property>
      <property name="text">
       <string>Dia Anivers√°rio</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>440</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #6A6A6A;</string>
      </property>
      <property name="text">
       <string>M√™s Anivers√°rio</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnOk">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>590</y>
        <width>355</width>
        <height>47</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>355</width>
        <height>47</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>355</width>
        <height>47</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border: none;
	background-color: #48A7FF;
	color: #fff;
	font-size: 16px;
	font-weight: 600;
}

QPushButton:hover{
	background-color: #4DBCFF;
}</string>
      </property>
      <property name="text">
       <string>SALVAR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>530</y>
        <width>174</width>
        <height>47</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>174</width>
        <height>47</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>47</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border: none;
	background-color: #48A7FF;
	color: #fff;
	font-size: 16px;
	font-weight: 600;
}

QPushButton:hover{
	background-color: #4DBCFF;
}</string>
      </property>
      <property name="text">
       <string>EDITAR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRemove">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>530</y>
        <width>174</width>
        <height>47</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>174</width>
        <height>47</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>171</width>
        <height>47</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border: none;
	background-color: #FF4848;
	color: #fff;
	font-size: 16px;
	font-weight: 600;
}

QPushButton:hover{
		background-color: #FF6060;
}</string>
      </property>
      <property name="text">
       <string>APAGAR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnBackEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: none;
	border: none;
}

QPushButton:hover{
	background-color: #E1E1E1;
	border-radius: 7px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="src.qrc">
        <normaloff>:/src/img/iconBack.png</normaloff>:/src/img/iconBack.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
